apis/management.cattle.io/v3/authn_types.go

```diff
@@ -349,6 +352,11 @@ type OKTAConfig struct {
 	SamlConfig `json:",inline" mapstructure:",squash"`
 }
 
+type ShibbolethConfig struct {
+	SamlConfig `json:",inline" mapstructure:",squash"`
+	LdapFields `json:",inline" mapstructure:",squash"`
```

Comment:
These ldap fields should not be inlined. They should be under their own ldapSearchConfig property.
The presence or absence of this optional property should indicate whether the user has configured this feature or not.